ci/integrationtests/task.yaml

```diff
@@ -71,10 +71,9 @@ spec:
 
         echo "> Installing dependencies"
         apk update && apk add make
-        pip install pipenv
```

Comment:
- why do you want to get rid of pipenv ?
- where are test dependencies installed ?
- it looks like you want to switch from pipenv to pip only for automated test run, correct ?
I wouldn't like to do this for developing integration tests since to my experience installing python dependencies without using a virtual environment is not a good idea

Reply:
Yes, I would like to remove pipenv for automated testruns, why:
- I didn't manage to install pipenv in the cc/job-image image, neither in debian:bullseye
- the tests "only" need the `scp` module in addition to what is available already with the standard cc/job-image
- eventually we should have a docker image containing all required dependencies instead of adding them at runtime and thus unnecessarily prolonging each and every testrun.
- I don't mind/care what is being used for local development, be it a virtual env or be it a docker container, ideally it should be the same dep mgmt as in the testruns, but as said, I did not manage to install pipenv with the above mentioned images
- ideally you can provide us with an image (or dockerfile) where the tests can be executed in the way you prefer, i.e. with pipenv

Reply:
which base image do you typically use for docker images ?

Reply:
We regularly use `eu.gcr.io/gardener-project/cc/job-image`.

Reply:
where is the source of this image ?

Reply:
I guess https://github.com/gardener/cc-utils/blob/master/Dockerfile, @ccwienk is this correct?

Reply:
ping @msohn, how should we proceed? Merge this PR or wait for other suggestions or docker images from you?